canopeum_backend/canopeum_backend/management/commands/initialize_database.py

```diff
@@ -349,7 +367,7 @@ def create_canopeum_site(self):
             name="First Batch",
             site=site,
             created_at=timezone.now(),
-            size=100,
+            size="100",
```

Comment:
Is size intended to be a string here? By the name alone it sounds like it might be intended to be an int.

Reply:
I opened an issue: #139